src/api/app/policies/bs_request_policy.rb

```diff
@@ -38,6 +38,15 @@ def decline_request?
     !(author? || record.is_source_maintainer?(user))
   end
 
+  def accept_request?
+    is_target_maintainer = record.is_target_maintainer?(user)
```

Comment:
Why assign this to a variable?

Reply:
Because  I copied the line from somewhere else, let me change it